scala/private/phases/phase_java_wrapper.bzl

```diff
@@ -47,3 +47,29 @@ def _phase_java_wrapper(
         args,
         wrapper_preamble,
     )
+
+def write_java_wrapper(ctx, args = "", wrapper_preamble = ""):
```

Comment:
Can we combine `_phase_java_wrapper` and `write_java_wrapper`? Or at least make `write_java_wrapper` private?

Reply:
fixed